1436790/1438462: Part 10: 20 files with MSDN link fixes.

DennisLee-DennisLee opened the pull request:
Contributes to PR #[2037](https://github.com/dotnet/docs/issues/2037) and applies to the following work items:
- [User Story 1436790: Content Update: update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1436790)
- [Task 1438462: Update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1438462)

DennisLee-DennisLee committed:
20 files with MSDN link fixes.

DennisLee-DennisLee committed:
Fixed 16 files worth of AdventureWorks links.

mairaw commented on docs/framework/data/adonet/ef/language-reference/method-based-query-syntax-examples-navigating-relationships.md:
```diff
@@ -47,5 +47,5 @@ Navigation properties in the [!INCLUDE[adonet_ef](../../../../../../includes/ado
   
 ## See also
-- [Navigation Properties](https://msdn.microsoft.com/library/41e1e6b9-8a57-467d-99d9-1857d2ca2ea5)
+- [Navigation Properties](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/bb738520(v=vs.100))
```
There's a second topic that uses the same link. Make sure we change that one as well to use this same link below.

```suggestion
- [Relationships, navigation properties and foreign keys](/ef/ef6/fundamentals/relationships)
```

mairaw reviewed (approved):
Looks good @DennisLee-DennisLee. Left just one comment and then we can merge

mairaw committed:
Update docs/framework/data/adonet/ef/language-reference/method-based-query-syntax-examples-navigating-relationships.md

Accepting Maira's suggestion. The other occurrence of the link is in part 9 (PR #10318) and has been fixed through commit c0d7ee5.

Co-Authored-By: DennisLee-DennisLee <v-dele@microsoft.com>
